core/plugin/zplugin.go

```diff
@@ -48,6 +48,4 @@ import (
 	_ "github.com/coredns/coredns/plugin/trace"
 	_ "github.com/coredns/coredns/plugin/transfer"
 	_ "github.com/coredns/coredns/plugin/whoami"
-	_ "github.com/coredns/federation"
-	_ "github.com/openshift/coredns-mdns"
```

Comment:
We need to keep coredns-mdns for KNI (bare metal).

Reply:
Whoops. Pushing a fix. Thanks!

Reply:
(Note to self: Triple check upstream commit contents before cherry-picking)

Reply:
Fixed :grin: